autodispose-rxlifecycle-interop/src/main/java/com/ubercab/autodispose/rxlifecycleinterop/RXLifeCycleInterop.java

```diff
@@ -0,0 +1,54 @@
+package com.ubercab.autodispose.rxlifecycleinterop;
+
+import com.trello.rxlifecycle2.LifecycleProvider;
+import com.trello.rxlifecycle2.OutsideLifecycleException;
+import com.trello.rxlifecycle2.RxLifecycle;
+import com.uber.autodispose.AutoDispose;
+import com.uber.autodispose.LifecycleEndedException;
+import com.uber.autodispose.ScopeProvider;
+import io.reactivex.Maybe;
+
+/**
+ * Interop class for {@link RxLifecycle}. It provides static utility methods to convert {@link
+ * LifecycleProvider} to {@link ScopeProvider}
+ * <p> There are several static utility converter
+ * methods such as {@link #bindLifecycle(LifecycleProvider)} for {@link
+ * LifecycleProvider#bindToLifecycle()} and {@link #bindUntilEvent(LifecycleProvider, Object)} for
+ * {@link LifecycleProvider#bindUntilEvent(Object)} </p>
+ *
+ * <em>Unlike {@link AutoDispose}, {@link RxLifecycle} treats the {@link OutsideLifecycleException}
```

Comment:
Nit: remove the autodispose bit at the beginning

also, only make the beginning emphasized. Do something like:

`<em>Note:</em> RxLifecycle treats...`